src/sentry/static/sentry/app/utils/analytics.tsx

```diff
@@ -48,22 +48,22 @@ export const trackAnalyticsEvent = options =>
  * @param {String} options.eventKey The string key of the event to track
  * @param {...Object} options.data The parameters of the event to track
  */
-export const trackAdhocEvent = options =>
+export const trackAdhocEvent = (options: {[key: string]: any}) =>
```

Comment:
Is the value actually any? Or would `number | string | null` cover the cases we want to allow?

Reply:
I'm not sure if there is a restriction in what types we can send to reload. I assumed we can also send arrays / objects as well though hence `any`.

Reply:
https://github.com/getsentry/reload/blob/133887e8856746d9e7ce2f2a1cc3205d38b412a1/reload_app/events.py

Looks like number, string, boolean

Reply:
Updated